managedtenants/data/selectorsyncset.yaml.j2

```diff
@@ -179,8 +187,17 @@ items:
   spec:
     clusterDeploymentSelector:
       matchLabels:
+        {% if ADDON.metadata.get('syncsetMigration') in ["step 3 - change SSS label", "step 4 - enable syncset", "step 5 - migration complete", "rollback ocm"] %}
```

Comment:
As discussed, we don't need to change this section.

Reply:
addressed :)